parlai/agents/legacy_agents/seq2seq/seq2seq_v0.py

```diff
@@ -112,14 +112,14 @@ class Seq2seqAgent(Agent):
     """
 
     OPTIM_OPTS = {
-        'adadelta': optim.Adadelta,
-        'adagrad': optim.Adagrad,
+        'adadelta': optim.Adadelta,  # type: ignore
+        'adagrad': optim.Adagrad,  # type: ignore
         'adam': optim.Adam,
```

Comment:
is there a reason we don't ignore Adam's type?

Reply:
Adam has types!